<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeerImageWidgetForm</class>
 <widget class="QWidget" name="SeerImageWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seer Image Viewer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="formatComboBox">
       <item>
        <property name="text">
         <string>RGBA8888</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RGB888</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="widthLineEdit">
       <property name="toolTip">
        <string>The image width (pixels)</string>
       </property>
       <property name="placeholderText">
        <string>width</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="heightLineEdit">
       <property name="toolTip">
        <string>The image height (pixels)</string>
       </property>
       <property name="placeholderText">
        <string>height</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="redrawToolButton">
       <property name="toolTip">
        <string>Redraw image.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/view-refresh.svg</normaloff>:/seer/resources/RelaxLightIcons/view-refresh.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QImageViewer" name="imageViewer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
     <property name="text" stdset="0">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="statusLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QImageViewer</class>
   <extends>QWidget</extends>
   <header>QImageViewer.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>formatComboBox</tabstop>
  <tabstop>widthLineEdit</tabstop>
  <tabstop>heightLineEdit</tabstop>
  <tabstop>redrawToolButton</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
